### ISSUE TITLE
 Remove noindex from documentation sites of previous releases.

### ISSUE THREAD

**Issue by rhalff** [balderdashy/sails#6884]
**Node version**: 
**Sails version** v0.12: 
**ORM hook version** _(sails-hook-orm)_: 
**Sockets hook version** _(sails-hook-sockets)_: 
**Organics hook version** _(sails-hook-organics)_: 
**Grunt hook version** _(sails-hook-grunt)_: 
**Uploads hook version** _(sails-hook-uploads)_: 
**DB adapter & version** _(e.g. sails-mysql@5.55.5)_: 
**Skipper adapter & version** _(e.g. skipper-s3@5.55.5)_: 
<hr/>

Hi I've noticed the documentation sites of older versions of sails are using:
```html
 <meta name="robots" content="noindex">
```
As a result hardly anything about previous versions shows up on google. e.g. [0.12.sailsjs.com](https://www.google.com/search?hl=en&as_q=sails&as_epq=&as_oq=&as_eq=&as_nlo=&as_nhi=&lr=&cr=&as_qdr=all&as_sitesearch=0.12.sailsjs.com&as_occt=any&safe=images&as_filetype=&as_rights=)

That's a pity because in general the documentation seems to be very well maintained.

Only after visiting https://github.com/balderdashy/sails-docs it becomes clear there is still documentation available.

**Comment by sailsbot** [comment]
@rhalff Thanks for posting!  We'll take a look as soon as possible.

In the mean time, there are a few ways you can help speed things along:
 - look for a workaround.  _(Even if it's just temporary, sharing your solution can save someone else a lot of time and effort.)_
 - tell us why this issue is important to you and your team.  What are you trying to accomplish?  _(Submissions with a little bit of human context tend to be easier to understand and faster to resolve.)_
 - make sure you've provided clear instructions on how to reproduce the bug from a clean install.
 - double-check that you've provided all of the requested version and dependency information.  _(Some of this info might seem irrelevant at first, like which database adapter you're using, but we ask that you include it anyway.  Oftentimes an issue is caused by a confluence of unexpected factors, and it can save everybody a ton of time to know all the details up front.)_
 - read the [code of conduct](https://sailsjs.com/documentation/contributing/code-of-conduct).
 - if appropriate, ask your business to [sponsor your issue](https://sailsjs.com/support).   _(Open source is our passion, and our core maintainers volunteer many of their nights and weekends working on Sails.  But you only get so many nights and weekends in life, and stuff gets done a lot faster when you can work on it during normal daylight hours.)_
 - let us know if you are using a 3rd party plugin; whether that's a database adapter, a non-standard view engine, or any other dependency maintained by someone other than our core team.  _(Besides the name of the 3rd party package, it helps to include the exact version you're using.  If you're unsure, check out [this list of all the core packages we maintain](https://sailsjs.com/architecture).)_ 
<hr/>

Please remember: never post in a public forum if you believe you've found a genuine security vulnerability.  Instead, [disclose it responsibly](https://sailsjs.com/security).

For help with questions about Sails, [click here](http://sailsjs.com/support).

**Comment by madisonhicks** [comment]
Hi, @rhalff!

We set `content=noindex` there because the v0.12 site is largely just a copy of the v1 site, and we don't want Google indexing duplicate content. Ideally, it might be nice to have select v0.12 docs able to be indexed. In the meantime, a good workaround might be to view the docs in GitHub and use the search feature.
